zulipterminal/ui_tools/buttons.py

```diff
@@ -287,6 +289,12 @@ def __init__(
             show_function=self._narrow_with_compose,
             count=count,
         )
+
+        if realm_enable_guest_user_indicator and is_guest:
+            self.suffix_style = "current_user"
+            self.suffix_text = "(guest)"
+            self.update_widget()
```

Comment:
the web app does not contain '(you)' for the current user but on hover it overrides the guest suffix take a look at the screenshot attached, Ill put this down the code so that '(you)' gets overridden if you want me to add guest for the active user as well.  
<img width="469" alt="Screenshot 2024-01-16 at 7 42 13 PM" src="https://github.com/zulip/zulip-terminal/assets/85862184/0513a565-4284-47c6-94bd-e79fc17fd2e5">

Reply:
Yeah, this will be part of the challenge with how to render this in the user list. I'm not sure there's an obvious solution given our platform constraints, but isolating these changes to their own commit will make this easier to examine separately.